src/c++/perf_analyzer/genai-perf/genai_perf/parser.py

```diff
@@ -431,6 +438,46 @@ def get_extra_inputs_as_dict(args: argparse.Namespace) -> dict:
     return request_inputs
 
 
+def _parse_compare_args(subparsers) -> argparse.ArgumentParser:
+    compare = subparsers.add_parser(
+        "compare", help="Generate plots that compare multiple runs."
+    )
+    compare_group = compare.add_argument_group("Compare")
+    mx_group = compare_group.add_mutually_exclusive_group(required=False)
+    mx_group.add_argument(
+        "--config",
+        type=Path,
+        default=None,
+        help="The path to the YAML file that specifies plot configurations for "
+        "comparing multiple runs.",
+    )
+    mx_group.add_argument(
+        "-f",
+        "--files",
+        nargs="+",
+        default=[],
+        help="List of paths to the profile export JSON files. Users can specify "
+        "this option instead of the `--config` option if they would like "
+        "GenAI-Perf to generate default plots as well as initial YAML config file.",
+    )
+    compare.set_defaults(func=compare_handler)
+    return compare
+
+
+### Handlers ###
+
+
+def handler(args, extra_args):
```

Comment:
```suggestion
def profile_handler(args, extra_args):
```

Reply:
Updated

Reply:
I don't want to add any more work for you, but if you do make more changes, it would be nice to make this `handle_profile` to be [consistent with Triton CLI](https://github.com/triton-inference-server/triton_cli/blame/58ba74c4156b24541fd1fd6f462fe17f019d48d9/src/triton_cli/parser.py#L406) and make any integration with it more straightforward.

If we do not do it as part of this ticket, the next person to add handlers should move to this approach to set the standard IMO.